### DISCUSSION TITLE
help needed to run axolotl on sagemaker

### DISCUSSION THREAD

**Discussion by sandys** [axolotl-ai-cloud/axolotl#839] [General]
hi
we have been using sagemaker to run our llama2 finetunes. We want to start using axolotl. Can axolotl be used on sagemaker ?

we have been looking at https://github.com/szamarin/axolotl-sagemaker-training/blob/main/launch_training_job.ipynb as an example. but struggling with it. is there an official way to use axolotl on sagemaker?

**Comment by NanoCode012** [comment]
Very very late reply but since transformers support Sagemaker, it should just work? I've seen folks mention using this on our discord.
